DatadogCore/Tests/Datadog/RUM/RUMMonitorTests.swift

```diff
@@ -1332,7 +1332,7 @@ class RUMMonitorTests: XCTestCase {
     }
 
     private func verifyGlobalAttributes(in matchers: [RUMEventMatcher]) {
-        for matcher in matchers {
+        for matcher in matchers.filterTelemetry() {
```

Comment:
Now that the telemetry is injected, we are getting events during the initialisation of the Features. Previously, we were loosing those because `DD.telemetry` was not yet set.